src/DataBox/DataBox/DataBox.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <PsModuleName>DataBox</PsModuleName>
+    <AssemblyName>Microsoft.Azure.PowerShell.Cmdlets.DataBox</AssemblyName>
+    <RootNamespace>Microsoft.Azure.PowerShell.Cmdlets.DataBox</RootNamespace>
+  </PropertyGroup>
+
+  <Import Project="$(MSBuildThisFileDirectory)..\..\Az.props" />
+
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.Management.DataBox" Version="1.0.0" />
+    <PackageReference Include="Microsoft.Azure.Management.Storage" Version="12.0.0" />
```

Comment:
You cannot take a dependency on storage management dll - this conflicts with the dependency in Az.Storage.  Instead, use the common storage management client in this package that you alreadyu have a dependency on: https://github.com/Azure/azure-powershell/blob/master/tools/Common.Netcore.Dependencies.targets#L18